common/app/views/fragments/inlineJSBlocking.scala.html

```diff
@@ -59,6 +58,11 @@
     }
 
 
+    // ************* ANALYTICS *************
+
+    // Ophan pageview ID and browser ID are needed by Google Analytics, which runs sync at the bottom of the page
+    @InlineJs(ophanConfig().body, "ophanConfig.js")
```

Comment:
not with with being inlined, but can it go to the non-blocking? we don't do anything with ophan until domcontentlaoded fires anyway, so it should be fine. blocking is really for stuff we need done before paint can begin.